### ISSUE TITLE
Symfony versions supported

### ISSUE THREAD

**Issue by franmomu** [sonata-project/dev-kit#510]
The thing is that right now sonata packages support these symfony versions:

```
"^2.8 || ^3.2 || ^4.0",
```
Looking at [Symfony Roadmap](https://symfony.com/releases), version `2.8` is about to be unmaintained, `3.2` is around 2 years unmaintained and `4.0` almost a year unmaintained and `4.4`(LTS)/`5.0` is about to be released. 

That being said, I think the constraints could be raised to at least `^3.4` and `^4.2` and I don't know if there is plan to drop `2.8` in the next major version or could it be done in this one.

**Comment by greg0ire** [comment]
The rules are described [here](https://github.com/sonata-project/dev-kit/blob/master/project/CONTRIBUTING.md#dependency-changes)

Tl;DR: You can bump to `^3.4` and `^4.2`, but we can refuse (personally, I probably will not refuse :) )

**Comment by franmomu** [comment]
They are all merged 😌

**Comment by greg0ire** [comment]
Great job! Thanks a ot!
